src/ServicePulse.Host/app/modules/monitoring/js/monitored_endpoints.controller.js

```diff
@@ -87,18 +92,21 @@
             });
 
             endpointsFromScSubscription =
-                Rx.Observable.interval(5000)
+                Rx.Observable.interval(5000).startWith(0)
                 .flatMap(function(i) {
-                    return Rx.Observable.fromPromise(serviceControlService.getExceptionGroups('Endpoint Name', null));
+                    return Rx.Observable.fromPromise(serviceControlService.getExceptionGroups('Endpoint Name', ''));
                 }).selectMany(function(endpoints) {
                     return endpoints.data;
                 }).subscribe(function (endpoint) {
                     var index = $scope.endpoints.findIndex(function(item) { return item.name === endpoint.title });
                     if (index >= 0) {
+                        $scope.endpoints[index].serviceControlId = endpoint.id;
                         $scope.endpoints[index].errorCount = endpoint.count;
                     } else {
                         $scope.endpoints.push({ name: endpoint.title, errorCount: endpoint.count, isConnected: false, isServiceControlOnly: true, serviceControlId: endpoint.id });
                     }
+
+                    $scope.$apply();
```

Comment:
Is this needed for _every_ subscription update, or only the first?

Reply:
This is actually not needed at all. Removed.